### PR TITLE
terms & conditions and privacy policy

### PR DESCRIPTION
**Pull request by pvighi**
- Changed format
- fixed australia link
- added links to t&c and privacy policy in the disclamer section so PayPal users can see it

legalbefore
legal

### CODE REVIEW

**Comment by pvighi** [comment]
We still need to fix the fact that changing countries from the dropdown doesn't update the non-react parts of the contribution page so the terms and conditions link will not be updated.

**Comment by alexduf** [comment]
I think we should fix the problem you are mentioning by using react at the bottom of the page as well. (but that's for another PR)

File: app/views/fragments/giraffe/legalNotice.scala.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@(termsAndConditionsLink: String)()
+    <p>By proceeding, you are agreeing to our <a href="@termsAndConditionsLink"> Terms & Conditions</a>
```
Comment (alexduf) [inline app/views/fragments/giraffe/legalNotice.scala.html]: `&` => `&amp;`

**Comment by AWare** [comment]
:+1:

**Comment by alexduf** [comment]
:+1:

**Pull request merged**